### ISSUE TITLE
Support for phpstan-specific annotations in UselessVariable sniff

### ISSUE THREAD

**Issue by janlanger** [slevomat/coding-standard#789]
Please add support for phpstan specific annotation `@phpstan-var` so variable in not detected as useless.

```
/** @phpstan-var T $instance */
$instance = $enumClass::get($randomValue);
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/b648acffe1b688396b4816429ec4df96558699e1
